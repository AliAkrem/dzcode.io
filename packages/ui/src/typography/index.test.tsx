import { Typography } from ".";
import { render } from "@testing-library/react";

it("should render Typography component", () => {
  const { container } = render(<Typography />);
  expect(container).toMatchSnapshot();
});
