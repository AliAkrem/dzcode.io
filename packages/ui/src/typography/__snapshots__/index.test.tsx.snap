// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Typography component 1`] = `
<div>
  <p
    class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
  />
</div>
`;
